_data/navigation.yml

```diff
@@ -340,3 +340,73 @@ es:
       twitter_impersonation:
         title: "Twitter impersonation"  # XXX
         url: "/en/twitter-impersonation"  # XXX
+tr:
+  home:
+    title: "Bitcoin Core"
+    url:   "/tr/"
+  about:
+    title: "Hakkında"
+    url:   "/tr/about/"
+  download:
+    title: "İndir"
+    url: "/tr/download/"
+  blog:
+    title: "blog"
+    url:   "/en/blog/"
+  releases:
+    title: "Versiyonlar"
+    url:   "/tr/releases/"
+  faq:
+    title: "FAQs"
```

Comment:
this was removed

Reply:
Thanks for the info, I'll revise it now...

Reply:
Thanks. Removed faq nav (8985a37)